test/middleware/spec.back-link.js

```diff
@@ -1,11 +1,18 @@
-var backLinks = require('../../lib/middleware/back-links');
-
-var util = require('util'),
+var proxyquire = require('proxyquire'),
+    util = require('util'),
     EventEmitter = require('events').EventEmitter;
 
+var getBackLinks = function (previousSteps) {
+    return proxyquire('../../lib/middleware/back-links', {
+        '../util/helpers': {
+            getPreviousSteps: sinon.stub().returns(previousSteps)
+        }
+    });
+};
+
```

Comment:
The new helpers file is a dependency of the back links middleware and the `previousSteps` method is referenced during the `require`. If you were to just require the helpers you could stub the method. This could then default to return an empty array (the default of the helper) and the tests wouldn't need littering with an empty function call.